TestCases/TestCase.py

```diff
@@ -76,10 +76,10 @@ def __init__(self,tag_in):
     def run_test(self):
 
         print('==================== Start Test: %s ===================='%self.tag)
-        passed       = True
+        passed       = False
         exceed_tol   = False
         timed_out    = False
-        iter_missing = False
```

Comment:
So help me god, that this didn't create a bunch of silently failing reg-tests over the years :pray:

Reply:
Seems to be fine...